.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
```

Comment:
Does it need to be daily?

Reply:
We can change that! Open to suggestions :slightly_smiling_face:

Reply:
Fortnightly would be ideal for me. Since I do package maintenance on some other projects on that schedule.

Reply:
Annoyingly fortnightly isn't an option :disappointed: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval
I've set it to monthly for now